nf_core/lint/schema_lint.py

```diff
@@ -2,6 +2,22 @@
 
 import logging
 import nf_core.schema
+import jsonschema
+
+
+def remove_required_fields(schema):
+    """ Remove all required fields from a schema """
+    for group_key in schema["definitions"].keys():
+        group = schema["definitions"][group_key]
+        group_keys = list(group.keys())
+        if "required" in group_keys:
+            required_params = group["required"]
+            for rp in required_params:
+                schema["definitions"][group_key]["properties"].pop(rp)
```

Comment:
Do you need to remove the actual property as well? I think just removing the `required` attribute should be sufficient. The property will just be ignored if there is no value for it.

Reply:
yeah that's actually a good point ....thanks!

Reply:
alright now it's only 3 lines of code 🙄